mail/dovecot2/Portfile

```diff
@@ -105,150 +132,95 @@ variant libstemmer description {Use libstemmer for full-text search} {
-    configure.args-append       --with-mysql
+variant mysql57 \
+    conflicts mysql8 mariadb mariadb10.1 mariadb10.2 percona \
+    description "Enable MySQL 5.7 (MySQL) support" {
+        depends_lib-append          port:mysql57
+        configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql57/bin/mysql_config
+        configure.args-append       --with-mysql
 }
 
-variant mysql56 \
-    conflicts mysql5 mysql51 mysql55 mariadb percona \
-    description "Enable MySQL 5.6 support" {
-    depends_lib-append          port:mysql56
-    configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql56/bin/mysql_config
-    configure.args-append       --with-mysql
+variant mysql8 \
+    conflicts mysql57  mariadb mariadb10.1 mariadb10.2 percona \
+    description "Enable Mysql 8 (MySQL) support" {
+        depends_lib-append          port:mysql8
+        configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql8/bin/mysql_config
+        configure.args-append       --with-mysql
 }
 
-variant mysql55 \
-    conflicts mysql5 mysql51 mysql56 mariadb percona \
-    description "Enable MySQL 5.5 support" {
-    depends_lib-append          port:mysql55
-    configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql55/bin/mysql_config
-    configure.args-append       --with-mysql
+variant mariadb \
+    conflicts mysql57 mysql8  mariadb10.1 mariadb10.2 percona \
+    description "Enable MariaDB 5 (MySQL) support" {
+        depends_lib-append          port:mariadb
+        configure.env-append        MYSQL_CONFIG=${prefix}/lib/mariadb/bin/mysql_config
+        configure.args-append       --with-mysql
 }
 
-variant mysql51 \
-    conflicts mysql5 mysql55 mysql56 mariadb percona \
-    description "Enable MySQL 5.1 support" {
-    depends_lib-append          port:mysql51
-    configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql51/bin/mysql_config
-    configure.args-append       --with-mysql
+variant mariadb10.1 \
+    conflicts mysql57 mysql8 mariadb  mariadb10.2 percona \
+    description "Enable MariaDB 10.1 (MySQL) support" {
+        depends_lib-append          port:mariadb-10.1
+        configure.env-append        MYSQL_CONFIG=${prefix}/lib/mariadb-10.1/bin/mysql_config
+        configure.args-append       --with-mysql
 }
 
-variant mysql5 \
-    conflicts mysql51 mysql55 mysql56 mariadb percona \
-    description "Enable MySQL 5.1 support" {
-    depends_lib-append          port:mysql5
-    configure.env-append        MYSQL_CONFIG=${prefix}/lib/mysql5/bin/mysql_config
-    configure.args-append       --with-mysql
+variant mariadb10.2 \
+    conflicts mysql57 mysql8 mariadb mariadb10.1 percona \
+    description "Enable MariaDB 10.2 (MySQL) support" {
+        depends_lib-append          port:mariadb-10.2
+        configure.env-append        MYSQL_CONFIG=${prefix}/lib/mariadb-10.2/bin/mysql_config
+        configure.args-append       --with-mysql
 }
 
+
 variant no_startupitem description {Do not install a launchd plist} {}
```

Comment:
This variant is no longer needed. Users can just set `startupitem_install no` in their macports.conf.

Reply:
removed